homeassistant/components/sensor/rmvtransport.py

```diff
@@ -78,6 +78,19 @@
 
     session = async_get_clientsession(hass)
 
+    async_setup_platform.available = None
```

Comment:
This is weird. This is a sign that we should refactor the platform. Perhaps have single instance the does the api interaction. This could hold data for multiple departures / entities.